lib/manageiq/password.rb

```diff
@@ -139,8 +139,14 @@ def self.unwrap(str)
       str.match(REGEXP_START_LINE)&.public_send(:[], 1)
     end
 
-    def self.store_key_file(filename, key)
+    def self.store_key_file(filename, key, groupname = nil)
       File.write(filename, key.to_h.to_yaml)
+      File.chmod(0440, filename)
```

Comment:
These can be a single method call

```suggestion
      File.write(filename, key.to_h.to_yaml, :perm => 0440)
```